project/Common.scala

```diff
@@ -39,7 +39,7 @@ object Common extends AutoPlugin {
     projectInfoVersion := (if (isSnapshot.value) "snapshot" else version.value),
     crossVersion := CrossVersion.binary,
     crossScalaVersions := Dependencies.ScalaVersions,
-    scalaVersion := Dependencies.Scala213,
+    scalaVersion := Dependencies.Scala212,
```

Comment:
Why? In many other projects we have changed to 2.13 as default, and run most tests with 2.13, but a `+Test/compile` or similar in validation checks.

Reply:
Alright, also a reasonable option, I can switch to that here as well.

Reply:
Done in [420f7fc](https://github.com/akka/akka-projection/pull/700/commits/420f7fc90f4e4c79153754a0c875e6b6b01efecb)

We already had cross-compile check as part of validation, not sure how 2.13 only code snuck through that earlier.